packages/next-common/components/fellowship/core/members/evidence.js

```diff
@@ -0,0 +1,87 @@
+import FieldLoading from "next-common/components/icons/fieldLoading";
+import FellowshipMemberInfoWrapper from "./infoWrapper";
+import FellowshipMemberInfoTitle from "./title";
+import { useEffect, useState } from "react";
+import { useContextApi } from "next-common/context/api";
+import { textEllipsis } from "next-common/utils";
+import getIpfsLink from "next-common/utils/env/ipfsEndpoint";
+import { create as createDigest } from "multiformats/hashes/digest";
+import { CID } from "multiformats";
+import { hexToU8a } from "@polkadot/util";
+import Tooltip from "next-common/components/tooltip";
+
+function useFellowshipCoreMemberEvidence(address) {
```

Comment:
How about `useSubFellowshipCoreMemberEvidence` considering we are subscribing a storage?